### DISCUSSION TITLE
Can I request an API token for the community edition of sysreptor

### DISCUSSION THREAD

**Discussion by MarcieLeeCoding** [Syslifters/sysreptor#329] [Q&A] [answered]
Reading the docs and looking around it seems like there is no way for the community edition to have an API token unless i am heavily missing something. If this is a feature only available for pro users I completely understand and will stumble my way through manually creating templates.

**Comment by MarcieLeeCoding** [comment, marked as answer]
nevermind i found the answer, it was hiding in plain sight the whole time; it's on the selfhosted instance at the same endpoint /users/self/apitokens i feel silly now

**Reply by aronmolnar** [reply]
No worries. Great you found it yourself.
Here's the UI guide for those who might need it :)

![image](https://github.com/user-attachments/assets/cadfbd46-284e-4318-aafd-db570dde464f)
